src/Ledger.Net/LedgerManager.cs

```diff
@@ -156,31 +156,57 @@ protected byte[] GetResponseDataPacket(byte[] data, int packetIndex, ref int rem
         #endregion
 
         #region Public Methods
-        public async Task<string> GetAddressAsync(uint coinNumber, uint account, bool isChange, uint index, bool showDisplay, AddressType addressType, bool isSegwit)
+        public async Task<string> GetAddressAsync(AddressType addressType, uint addressIndex)
```

Comment:
This method at least needs CoinNumber, and probably Account. There are multiple coins supported by the Bitcoin app, so assuming that we are looking for Bitcoin is not quite right. Also, Ethereum also supports at least two coins Ethereum: 60, and EthereumClassic: 61. 

I suppose we could keep this overload, but we'd need to add at least one more, and I try to keep downward pressure on overloads.